routes/accountRouter.js

```diff
@@ -287,12 +287,9 @@ routes.post("/createInvite", (req, res) => {
 routes.get("/createPlusInvite", (req, res) => {
   db.plusCodes.loadDatabase();
   if (req.user.toString() === process.env.DEVELOPER_USER) {
-    const plusCode =
-      Math.random().toString(36).slice(2) +
-      Math.random().toString(36).slice(2) +
-      Math.random().toString(36).slice(2) +
-      Math.random().toString(36).slice(2) +
-      Math.random().toString(36).slice(2);
+    let plusCode = "";
+    for(let i = 0; i < 5; i++) plusCode += Math.random().toString(36).slice(2);
```

Comment:
Im pushing an update for a new utility called 'generateString', when i push that, could you change this to use that utility?

Reply:
Yeah of course, let me know when that happens and I'll add it.